src/overlays/actors/ovl_En_Po_Composer/z_en_po_composer.h

```diff
@@ -2,16 +2,59 @@
 #define Z_EN_PO_COMPOSER_H
 
 #include "global.h"
+#include "objects/object_po_composer/object_po_composer.h"
+
+#define PO_COMPOSER_IS_FLAT(actor) ((actor)->params & 0x8000)
+#define PO_COMPOSER_4000(actor)    ((actor)->params & 0x4000)
+
+typedef enum {
+    /*  1 */ COMPOSER_CUEID_IDLE = 1,
```

Comment:
Just building off my previous comment, there's both `PO_COMPOSER_` (params above) and `COMPOSER_` here. Might be nice to make it all `POE_COMPOSER`, although you can leave it as `PO_COMPOSER_` if you want since that's the file name. It's nice to make these a more english/readable though